web/src/server_events_dispatch.js

```diff
@@ -597,32 +597,30 @@ export function dispatch_normal_event(event) {
                     stream_list.update_subscribe_to_more_streams_link();
                     break;
                 case "delete":
-                    for (const stream of event.streams) {
-                        const was_subscribed = sub_store.get(stream.stream_id).subscribed;
-                        const is_narrowed_to_stream = narrow_state.is_for_stream_id(
-                            stream.stream_id,
-                        );
-                        stream_data.delete_sub(stream.stream_id);
-                        stream_settings_ui.remove_stream(stream.stream_id);
-                        message_view_header.maybe_rerender_title_area_for_stream(stream);
```

Comment:
This should have been changed to pass `stream.stream_id` in the previous commit.

Reply:
you are commenting (by mistake I guess) on the changes (the red ones) that got removed not my changes.

Reply:
The first commit changes `maybe_rerender_title_area_for_stream` to accept ID instead of stream object. So, all the calls to this function should be updated in the first commit itself even when this code would be removed in the next commit.

Basically it is good to maintain a state that even if one commit from PR is merged and one is not, the app should work as expected, except when splitting changes makes it a lot easier to review changes.

Reply:
Yes, I get it now